go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/rancher/wrangler/v3
 
-go 1.23.3
+go 1.23.0
+
+toolchain go1.23.4
```

Comment:
This should be fine... we're on toolchain 1.23.3 for 1.32 at the moment but I suspect the next upstream release will bump that.

On the distros side we always use the same toolchain version as upstream, for any given patch release of Kubernetes:
https://github.com/kubernetes/kubernetes/blob/v1.32.0/build/dependencies.yaml#L118-L119